chain/core/api.py

```diff
@@ -179,6 +179,12 @@ def __init__(self, obj=None, queryset=None, data=None, request=None,
         self._request = request
         self._limit = limit or self.page_size
         self._offset = offset or 0
+        if self._queryset:
```

Comment:
Let's rename `_schema` so it doesn't conflict with the JSON schemas we use elsewhere. Maybe `_state` is better.